MC/bin/o2dpg_sim_workflow.py

```diff
@@ -205,6 +207,27 @@ def retrieve_sor(run_number):
 
     return int(SOR)
 
+# Recover mean vertex settings from external txt file
```

Comment:
I understood that the this should finally come from CCDB, right? So is this a tmp solution until we fetch directly from CCDB?

Reply:
Sorry, I forgot to answer here. 
Yes, this should come from CCDB, but if we have MC anchored to data reconstructed with the default mean vertex, and we want to have the simulation done with the real one, but still reconstructed as in data, then we cannot update the CCDB. This is why we need a way to pass the meanVtx by hand.